src/sentry/tsdb/snuba.py

```diff
@@ -278,6 +278,13 @@ def get_data(
             conditions += deepcopy(model_query_settings.conditions)
             # copy because we modify the conditions in snuba.query
 
+        orderby = []
+        if group_on_time:
+            orderby.append("-time")
+
+        if group_on_model and model_group is not None:
+            orderby.append(model_group)
```

Comment:
Worth just moving these to the `if` statements above?

Reply:
Could do, though I (subjectively) find this easier to read and the if statements above are in the reverse order of what we want (time condition first), so I'll leave it be.